app/settings/surveys/attribute-editor.html

```diff
@@ -79,6 +79,22 @@
              <label class="tgl-btn" for="switchnewattribute"></label>
           </div>
        </div>
+
+       <!-- start: media upload toggle to make caption an optional field-->
+       <div class="form-field switch" ng-show="canDisableCaption()">
+         <label translate="survey.disable_caption_field">Disable Caption field</label>
+         <p data-fieldgroup-target="field-caption-disabled-warning" class="init active" translate="survey.disable_caption_field_explanation" ng-show="!editAttribute.config.hasCaption">
+           <svg class="iconic">
```

Comment:
Does the SVG here show up? I think it'll get wiped out by the translate substitution

Reply:
No it doesn't and we didn't need the SVG there. That was a bad copy paste by me... I was copying the style from the required field and messed up.

Reply:
NOte: (I'll send this fix over after Jess's feedback)